docs/source/plugins.rst

```diff
@@ -5,30 +5,33 @@
 Plugins
 =======
 
-Plugins that come with the **IBM z/OS core collection** augment Ansible's core
+Plugins that come with the **IBM z/OS core collection** compliment Ansible's core
```

Comment:
`compliment` --> `complement`

Reply:
I seem to recall @richp405 having called this out somewhere else??

Reply:
Done